packages/javascript/bh-shared-ui/src/components/AssetGroupEdit/AssetGroupEdit.tsx

```diff
@@ -151,6 +161,7 @@ const FilteredMemberCountDisplay: FC<{
     const hasValidCount = !isLoading && !isError && count && count > 0;
 
     if (hasValidCount) {
+        handleAvailableNodeKind?.();
```

Comment:
Today I learned

Reply:
yess very handy with functions that are optional params!